messages/src/api.rs

```diff
@@ -103,6 +116,11 @@ pub enum ApplicationAPI {
     Version {
         version: String,
     },
+    RequestAvailableDags,
```

Comment:
Although I assume it wouldn't be a concern here, are there any cross-version compatibility concerns with changes to this enum? Like, is it append-only for new enum variants or whatever?

Reply:
A future enhancement along the path to SDK-ifying this software would be building more cross-version compatibility into these enums.